packages/mobile/src/navigator/NavigationService.ts

```diff
@@ -123,11 +127,20 @@ export async function ensurePincode(): Promise<boolean> {
     return false
   }
 
-  if (pincodeType !== PincodeType.CustomPin) {
+  if (pincodeType !== PincodeType.CustomPin && pincodeType !== PincodeType.PhoneAuth) {
     Logger.error(TAG + '@ensurePincode', `Unsupported Pincode Type ${pincodeType}`)
     return false
   }
 
+  if (pincodeType === PincodeType.PhoneAuth) {
+    try {
+      await getPincodeWithBiometry()
+      return true
+    } catch (error) {
+      Logger.error(`${TAG}@ensurePincode`, `Retrieve PIN by biometry error`, error)
```

Comment:
i've added a comment explaining! we need the flow to continue since we do not provide a fallback for the biometric auth - if face/touch id fails, then we need to continue in the function to activate the pincode input screen :)